### ISSUE TITLE
Creating Cluster in existing VPC fails on OTC

### ISSUE THREAD

**Issue by chrkl** [kubermatic/kubermatic#5292]
**User Story**

User tried to deploy a cluster to an existing VPC and Subnet on OTC. Both have been selected in the wizard during cluster creating.

Kubermatic created a new VPC instead of using the desired one.

The following error appeared in the events:
```
failed cloud provider init: failed to attach subnet to router: Bad request with: [PUT https://vpc.eu-de.otc.t-systems.com/v2.0/routers/8110c4f3-aaf2-432a-81ea-fe6870f5ef23/add_router_interface], error message: {"NeutronError":{"message":"Bad router request: Subnets or qr-ports in same network should be at same router.","type":"BadRequest","detail":""}}
```
8110c4f3-aaf2-432a-81ea-fe6870f5ef2 is the VPC that has been created by Kubermatic

**Acceptance criteria**

The user cluster should be deployed to the existing VPC and subnet.

**Comment by chrkl** [comment]
Works on "normal" OpenStack, problem is OTC specific.

**Comment by kdomanski** [comment]
fixed in #5334
